python_modules/dagster/dagster/core/storage/pipeline_run.py

```diff
@@ -203,12 +213,23 @@ def pipeline_run_from_storage(
     )
 
 
+@whitelist_for_serdes
+class PipelineTarget(NamedTuple):
+    name: str
+    mode: str
+
+
+@whitelist_for_serdes
+class JobTarget(NamedTuple):
+    name: str
+
+
 @whitelist_for_serdes(serializer=PipelineRunSerializer)
-class PipelineRun(
+class DagsterRun(
```

Comment:
Could we just call this `Run`? Is the worry that there would be overlap with step runs?  If that's the case, someone might think that `DagsterRun` is a parent of both step runs and pipeline/job runs.  Any reason not to go with `JobRun`?

Reply:
cc @alangenfeld . I personally get a bit scared by a name as simple and universal as `Run`.